src/components/ProductPage/ProductRipcord.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { ReactNode } from 'react'
+import styled from 'styled-components'
+import numeral from 'numeral'
+import { InputProps } from 'react-neu'
+
+import SimpleButton from 'components/SimpleButton'
+import { ProductPageSection } from './ProductPageLayouts'
+import useWallet from 'hooks/useWallet'
+import { useRipcord } from 'hooks/useRipcord'
+import { TokenDataProps } from './ProductDataUI'
+import {
+  Bitcoin2xFlexibleLeverageIndex,
+  Ethereum2xFlexibleLeverageIndex,
+} from 'constants/productTokens'
+
+interface ProductRipcordProps extends InputProps {
+  tokenData: TokenDataProps
+}
+
+const ProductRipcord: React.FC<ProductRipcordProps> = ({ tokenData }) => {
+  const { isMetamaskConnected } = useWallet()
+  const handleRipcord = useRipcord()
+
+  const formatLeverageMetric = (metricValue: number) =>
+    numeral(metricValue).format('0.00').toString().toUpperCase()
+
+  const realLeverage =
+    tokenData.components && tokenData.components.length > 0
+      ? Number(tokenData.components[0].percentOfSet || 0) / 100
+      : 2
+
+  const maxLeverage = Ethereum2xFlexibleLeverageIndex.symbol ? 2.3 : 2.2
```

Comment:
Could probably be improved upon, I don't love the hard-coding

Reply:
Can we make this a bit more explicit. Maybe set a variable at the top of the file or in a constants file.
`const BTC2xLeverageRatio = 2.2`
`const ETH2xLeverageRatio = 2.3`